RepoMan/RepoMan/Analysis/Scoring/CommentCountScorer.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Linq;
+using RepoMan.Repository;
+
+namespace RepoMan.Analysis.Scoring
+{
+    // It only counts as a comment if the word count is greater than 10 words.
+    class CommentCountScorer :
+        PullRequestScorer
+    {
+        public const string Label = "CommentCount";
+        public override string Attribute => Label;
+        public override double ScoreMultiplier => 20;
+        private readonly IWordCounter _wc;
+        private const int _wordCountFloor = 10;
+
+        public CommentCountScorer(IWordCounter wordCounter)
+        {
+            _wc = wordCounter ?? throw new ArgumentNullException(nameof(wordCounter));
+        }
+
+        public override int Count(PullRequestDetails prDetails)
+        {
+            var bodyCount = _wc.Count(prDetails.Body);
+            var commentCount = prDetails.AllComments
+                .Select(c => _wc.Count(c.Text))
+                .Count(l => l >= _wordCountFloor);
```

Comment:
My thought was that comments containing less than 10 words aren't meaningful, but I think I'm wrong on this. Questions can be powerful while only being just a few words.

Maybe I should just take out the limit.